src/discord/commands/LowestbinCommand.js

```diff
@@ -0,0 +1,76 @@
+/*
+ CREDIT: Idea by Aura
+ Discord: Aura#5051
+ Minecraft username: _aura
+*/
+const {SlashCommandBuilder} = require('@discordjs/builders')
+const fetch = require("axios");
+const {Index} = require("flexsearch");
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('lowestbin')
+        .setDescription('Display basic info about the client.')
+        .addStringOption(option =>
+            option.setName('item')
+                .setDescription('Item you wanna search the lowest bin of.')
+                .setRequired(true)),
+    permission: 0, // 0 = anyone, 1 = staff, 2 = owner/admin
+
+    async execute(clientInstance, interaction) {
+        await interaction.deferReply()
+
+        let itemname = interaction.options.getString("item")
+        let item = itemname.join(" ")
+
+        let foundItem = await findItem(itemname)
+
+        if (!foundItem) {
+            interaction.editReply(`${itemname} does not exist!`)
+        } else {
+            interaction.editReply(`${foundItem.name}'s lbin is ${localizedPrice(foundItem.lowestBin)}`)
+        }
+
+    }
+}
+
+let index = new Index()
+let cache
+let lastRetrieve = 0
+
+async function refreshCache() {
+    if (!cache || lastRetrieve + 300 * 1000 < new Date().getTime()) {
+        let data = await fetch('https://maro.skybrokers.xyz/api/auctions/all')
```

Comment:
this service has been shutdown. change the domain to skyblock.acebot.xyz
```suggestion
        let data = await fetch('https://skyblock.acebot.xyz/api/auctions/all')
```